src/cutoffs.jl

```diff
@@ -92,3 +92,48 @@ end
     potential(inter, r2, invr2, params) + (r - rc) * fc -
         potential(inter, cutoff.sqdist_cutoff, cutoff.inv_sqdist_cutoff, params)
 end
+struct CubicSplineCutoff{D,S,I}
```

Comment:
Add a docstring here. Also export it at the top of the file and refer to it on line 588 of `docs/src/docs.md`.